### ISSUE TITLE
cppwrap is broken with Java 8

### ISSUE THREAD

**Issue by ghost** [uw-loci/cppwrap-maven-plugin#2]
Please see the thread here: https://www.openmicroscopy.org/community/viewtopic.php?f=13&t=7729

It's working fine with Java 6 and Java 7, but is failing with Java 8.  Is this a known problem?  If not, updating the plugin to work with Java 8 would be much appreciated.

I did try updating jace to the latest version from http://search.maven.org/#search|ga|1|g%3A%22com.googlecode.jace%22 but this didn't work (missing class), so looks like it would require changes to the plugin itself to upgrade jace.  I don't know that upgrading jace would necessarily fix things, it might itself need work if the latest version is still broken and the problem is with jace rather than the plugin.

Kind regards,
Roger

**Comment by ctrueden** [comment]
Yes, the version of Jace we use is massively out of date. It would be awesome to use the latest release from Maven Central. But every time I tried in the past (admittedly years ago now), there were blockers preventing the upgrade. See also https://github.com/uw-loci/jar2lib/issues/4.

Agreed also that upgrading Jace might or might not fix the problem. But it seems like a good start regardless. That is: attempting to fix this issue _without_ upgrading Jace seems ill advised to me.
